src/cli/commands/graph.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import {Command, flags} from "@oclif/command";
+import mkdirp from "mkdirp";
+import os from "os";
+import path from "path";
+
+import {pluginNames} from "./common";
+
+const execDependencyGraph = require("../../tools/execDependencyGraph").default;
+
+export default class GraphCommand extends Command {
+  static description = "create the contribution graph for a repository";
```

Comment:
[optional] I'd like to see more documentation here. Specifically, it should document the output format, that the actual resultant graph is `outputdir/repoOwner/repoName/graph.json` and that the other json files are intermediary artifacts. Note that multiline descriptions are supported.

I mark this [optional] since it's reasonably easy to infer by inspecting the output directory, and this is likely to be a power user feature (regular users will prefer the method that starts sourcecred and calls this method under-the-hood)